include/apple2.h

```diff
@@ -83,7 +83,57 @@
 #define CH_CURS_LEFT    0x08
 #define CH_CURS_RIGHT   0x15
 
-#if !defined(__APPLE2ENH__)
+#if defined(__APPLE2ENH__)
+
+/* MouseText-based functions for boxes and lines drawing */
+void mt_chline(unsigned char length);
+void mt_cvline(unsigned char length);
+void mt_chlinexy(unsigned char x, unsigned char y, unsigned char length);
+void mt_cvlinexy(unsigned char x, unsigned char y, unsigned char length);
```

Comment:
Some nit picking: The (unusual) cc65 style for function prototypes is to have a space before the parameter list.